src/routes/+page.svelte

```diff
@@ -1,121 +1,155 @@
 <script>
-  import Header from '$lib/header.svelte'
+	import Header from '$lib/header.svelte';
 
-  let searchFor = ''
-  let wordDefinition
-  let isDefinitionFound = false 
-  let isSearchTextError = false
-  let isError = false
+	let searchFor = '';
+	let wordDefinition;
+	let isDefinitionFound = false;
+	let isSearchTextError = false;
+	let isError = false;
+	let wordCount = 0
 
-  let errorMessage
-  const ERR_INVALID_SEARCH_TEXT = "Please enter a single word to search for before clicking 'Get Definition'"
-  const ERR_API_ERROR = "An error occurred in the API. Error: "
+	let errorMessage;
+	const ERR_INVALID_SEARCH_TEXT =
+		"Please enter a single word to search for before clicking 'Get Definition'";
+	const ERR_API_ERROR = 'An error occurred in the API. Error: ';
 
-  const captureKeystroke = (event) => {
-    isSearchTextError = false
-    if (event.key === 'Enter') {
-      searchForDefinition()
-    }
-  }
+	/* Intercept presses of the Enter key */
+	const captureKeystroke = (event) => {
+		isSearchTextError = false;
+		if (event.key === 'Enter') {
+			countWords()
+			searchForDefinition();
+		}
+	};
 
-  const validateSearchWord = () => {
-    isError = false
-    if (searchFor === null || searchFor === undefined || searchFor === '') {
-      isError = true
-      errorMessage = ERR_INVALID_SEARCH_TEXT
-      return
-    }
+	// Count the number of words in the search box
+	const countWords = () => {
```

Comment:
You should use a string function searchFor.split(" ") so you can check for words, this function will convert the different words of the string into items in an array and that way you can use the length of the array to count the words

Reply:
Great idea. This should also be more performant. Issue resolve.